src/inputProcessor.js

```diff
@@ -10,11 +10,10 @@ function getEmptySvgDeclarationComputed() {
   return emptySvgDeclarationComputed
 }
 
-function getSource(svg) {
+function getSource(svg, options = {}) {
```

Comment:
Destructuring and providing default value here could simplify code a little bit:
```suggestion
function getSource(svg, { css = 'inline' }) {
```